src/Subscriber/FrontendSubscriber.php

```diff
@@ -31,21 +28,11 @@ public static function getSubscribedEvents(): array
 
     public function onHeaderLoaded(HeaderPageletLoadedEvent $event): void
     {
-        $config = $this->nostoConfigService->getConfigWithInheritance(
+        $config = $this->configProvider->toArray(
             $event->getSalesChannelContext()->getSalesChannelId(),
             $event->getSalesChannelContext()->getLanguageId(),
         );
         $nostoConfig = new Config($config);
         $event->getContext()->addExtension('nostoConfig', $nostoConfig);
```

Comment:
Only the config is required on every single page, the other ones only for search and navigation